src/boundary/maxwellian.cpp

```diff
@@ -243,10 +243,9 @@ void Maxwellian::generateTemplateCell(spatial_cell::SpatialCell &templateCell, R
                      for (uint vi = 0; vi < speciesParams[popID].nVelocitySamples; ++vi)
                         for (uint vj = 0; vj < speciesParams[popID].nVelocitySamples; ++vj)
                            for (uint vk = 0; vk < speciesParams[popID].nVelocitySamples; ++vk)
-                           {
                               avr += maxwellianDistribution(popID, rho, T, vxCell + vi * d_vx - Vx,
                                                             vyCell + vj * d_vy - Vy, vzCell + vk * d_vz - Vz);
-                           }
```

Comment:
Actually in principle for loops without a curly bracket are frowned upon in the Vlasiator style, only a few of such nested loops were accepted with a single pair of curly brackets. If something changes here, I'd rather have the three pairs than none at all.

Reply:
I will change this style accordingly.

Reply:
Thanks. :)